<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollisionFilterSetter</class>
 <widget class="QWidget" name="CollisionFilterSetter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>GroupID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>No Collide With: </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBox_10">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBox_11">
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="checkBox_12">
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QCheckBox" name="checkBox_13">
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QCheckBox" name="checkBox_14">
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QCheckBox" name="checkBox_15">
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_16">
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_17">
       <property name="text">
        <string>17</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkBox_18">
       <property name="text">
        <string>18</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="checkBox_19">
       <property name="text">
        <string>19</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="checkBox_20">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QCheckBox" name="checkBox_21">
       <property name="text">
        <string>21</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QCheckBox" name="checkBox_22">
       <property name="text">
        <string>22</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QCheckBox" name="checkBox_23">
       <property name="text">
        <string>23</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>-1 - Virtual, objects collide with nothing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0 - Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 - Body, Main part</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 - Body, Breasts and butts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 - Body, Arms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 - Body, Hands</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 - Body, Legs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 - Body, Feet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7 - Body, Pussy and asshole</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 - Body, Head</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9 - Body, Tail</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 - Body, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11 - Cloth, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 - Cloth, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13 - Cloth, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14 - Cloth, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 - Hair, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16 - Hair, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17 - Hair, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18 - Weapons, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19 - Weapons, Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20 - Reserved</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21 - Reserved</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22 - Reserved</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23 - Reserved</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
